pandas/tests/reshape/concat/test_datetimes.py

```diff
@@ -291,66 +307,87 @@ def test_concat_NaT_series_dataframe_all_NaT(self, tz1, tz2):
 
 
 class TestTimezoneConcat:
-    def test_concat_tz_series(self):
+    def test_concat_tz_series(self, unit, unit2):
```

Comment:
ditto i think not necessary here

Reply:
Yes indeed, the original issue was about dataframes but not series.